cloud_experts_tutorials/cloud-experts-consistent-egress-ip.adoc

```diff
@@ -191,21 +228,21 @@ demo-egress-ns    10.10.100.253   ip-10-10-156-122.ec2.internal   10.10.150.253
 demo-egress-pod   10.10.100.254   ip-10-10-156-122.ec2.internal   10.10.150.254
 ----
 
-== Test the egress IP rule
-
-=== Deploy a sample application
+== Testing the egress IP rule
```

Comment:
If you restructure the previous section as a procedure, this should then become a verification section.